### ISSUE TITLE
odd modification on standOff element

### ISSUE THREAD

**Issue by simonmeoni** [laurentromary/stdfSpec#1]
-  the type attribute must be mandatory on standOff

**Comment by laurentromary** [comment]
Not necessarily. I can foresee a use where a standOff element gathers all annotations from a project with sub-standOff elements being typed at lower level. We should at least make sure that standOff belongs to att.typed in all cases.

**Comment by simonmeoni** [comment]
the modification that we discussed early is present on #5 pull-request
